src/version.rs

```diff
@@ -300,16 +300,17 @@ pub fn dependencies(req: &mut Request) -> CargoResult<Response> {
 
 /// Handles the `GET /crates/:crate_id/:version/downloads` route.
 pub fn downloads(req: &mut Request) -> CargoResult<Response> {
+    let (offset, limit) = req.pagination(90, 100)?;
```

Comment:
I wouldn't call this offset and limit, since it's not used for either offset or limit. Not sure it makes sense to permit the `per_page` param here either.